alicloud/services/dds/client.go

```diff
@@ -0,0 +1,108 @@
+package dds
```

Comment:
I will push mongoDB developer to publish official go sdk. Please be patient.

Reply:
Please use official go sdk: https://github.com/aliyun/alibaba-cloud-sdk-go/tree/master/services/dds